### PR TITLE
Add ability to have custom content in OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by cblecker**
Not super pretty, but it works.

### CODE REVIEW

**Comment by cblecker** [comment]
/cc @cjwagner 
Hey Cole.. Will the approve plugin still process the file properly if we have comments in it?

**Comment by cjwagner** [comment]
> Hey Cole.. Will the approve plugin still process the file properly if we have comments in it?

Yes, we use a YAML parser to read the file. It should handle comments properly. Here is an OWNERS file that has a comment: https://github.com/kubernetes/steering/blob/4ba203d5dfa575aae1e230d411fc5ee2c0368923/OWNERS

**Comment by cblecker** [comment]
awesome! thanks @cjwagner

File: generator/app.go [hunk 1]
```diff
@@ -244,7 +269,7 @@ func createGroupReadme(groups []Group, prefix string) error {
 
 		outputPath := filepath.Join(outputDir, indexFilename)
 		readmePath := filepath.Join(baseGeneratorDir, templateDir, fmt.Sprintf("%s_%s", prefix, readmeTemplate))
-		if err := writeTemplate(readmePath, outputPath, true, group); err != nil {
+		if err := writeTemplate(readmePath, outputPath, "markdown", group); err != nil {
```
Comment (jamiehannaford) [inline generator/app.go]: instead of adding a new arg, could we compute this by reading the path extension?

Comment (cblecker) [reply generator/app.go]: `OWNERS_ALIASES` doesn’t have an extension :(

**Pull request merged**